src/modules/app/containers/RouterFragment.js

```diff
@@ -13,54 +13,80 @@ import PdfFetcherPage from 'routes/pdf-fetcher/containers/PdfFetcherPage'
 import MainDisclaimerPage from 'routes/main-disclaimer/components/MainDisclaimerPage'
 import LandingPage from 'routes/landing/containers/LandingPage'
 import CategoriesPage from 'routes/categories/containers/CategoriesPage'
+import I18nRedirect from 'modules/app/containers/I18nRedirect'
+import PropTypes from 'prop-types'
+import RouteConfig from '../RouteConfig'
 
 /**
  * todo: Test and document in WEBAPP-90
  * todo: Layouts should be set in each route
  */
 class RouterFragment extends React.Component {
+  static propTypes = {
+    routerConfig: PropTypes.instanceOf(RouteConfig).isRequired
+  }
+
+  static isLanguageCode (language) {
+    return language && language.length === 2
+  }
+
   render () {
+    const {routerConfig} = this.props
+    const matchRoute = (id) => routerConfig.matchRoute(id)
+
     /*
      * For routes inside a <React.Fragment /> the priority decreases with each element
      * So /disclaimer has higher priority than /:language -> '/disclaimer' resolves to /disclaimer
      */
     return <Fragment forRoute='/'>
       {/* Routes */}
       <React.Fragment>
+        {/* No language was provided to redirect to a specific language (e.g. the browsers language) */}
+        <Fragment forRoute='/'>
+          <I18nRedirect />
+        </Fragment>
 
         {/* Matches two or more arguments like /augsburg/de */}
         <Fragment forRoute='/:location/:language(/*)'>
```

Comment:
As discussed in lecture today, I'd really like these routes to be somehow referenced via js, (not put the string in manually), to deduplicate the definition of the routes... But I am also unsure how to do this. :/

Reply:
It's not possible without loosing a lot of flexibility.

Reply:
Came to this conclusion after several hours at least :D